earth_enterprise/src/server/wsgi/serve/snippets/util/dbroot_writer.py

```diff
@@ -109,7 +109,7 @@ def CreateEndSnippetProto(snippets_json,
   # We convert to string here to change from a bytes object to a python byte string.
   # We then remove the leading "b'" and the trailing "'" to ensure that swig handles
```

Comment:
I am guessing this comment should be changed since we no longer are stripping the lead b, etc...

Reply:
Yes